src/Take.Blip.Builder/LazyInput.cs

```diff
@@ -66,14 +66,22 @@ public class LazyInput
                     return envelopeSerializer.Serialize(Message);
                 }
 
-                return null;                
+                return null;
             });
         }
 
         public Message Message { get; }
-        
+
         public Document Content => Message.Content;
-        
+
+        public bool Analyzable {
```

Comment:
If not one is using this property outside the class, it doesn't need to be public. I suggest moving it to the constructor as a private field.